.kitchen.yml

```diff
@@ -44,11 +58,38 @@ platforms:
       extra_vars:
         ansible_python_interpreter: '/usr/local/bin/python'
 
+  - name: openbsd-6.1-amd64
+    driver:
+      box: trombik/ansible-openbsd-6.1-amd64
+      box_check_update: false
+    driver_config:
+      ssh:
+        shell: '/bin/sh'
+    provisioner:
+      extra_vars:
+        ansible_python_interpreter: '/usr/local/bin/python'
+
+  - name: openbsd-6.2-amd64
+    driver:
+      box: trombik/ansible-openbsd-6.2-amd64
+      box_check_update: false
+    driver_config:
+      ssh:
+        shell: '/bin/sh'
+    provisioner:
+      extra_vars:
+        ansible_python_interpreter: '/usr/local/bin/python'
+
   - name: ubuntu-14.04-amd64
     driver:
       box: trombik/ansible-ubuntu-14.04-amd64
       box_check_update: false
 
+  - name: ubuntu-16.04-amd64
+    driver:
+      box: trombik/ansible-ubuntu-14.04-amd64
```

Comment:
* typo: s/14/16/
* it should be added to `meta/main.yml` if `xenial` is supported by the PR.